### DISCUSSION TITLE
Github folders for each Discussion Group have been created.

### DISCUSSION THREAD

**Discussion by mutanthumb** [Digital-Preservation-Coalition/PythonStudyGroups#22] [Announcements]
Hi All-
I have created folders for each discussion group, feel free to add your code to your group's folder. This is not a requirement just an option. If anyone has any access issues or other questions please let me know!

https://github.com/Digital-Preservation-Coalition/PythonStudyGroups/tree/main/Cohort1

Thanks,
susan

**Comment by sefnyn** [comment]
Hi Susan,
When I try to push my code, it says:
remote: Permission to Digital-Preservation-Coalition/PythonStudyGroups.git denied to sefnyn.
Cheers, Nick

**Reply by mutanthumb** [reply]
Which group are you in Nick?

**Reply by mutanthumb** [reply]
I just added you as an outside collaborator, try it again and let me know if you still have issues.
